test/e2e/pods.go

```diff
@@ -155,6 +156,42 @@ var _ = Describe("Pods", func() {
 			},
 		})
 	})
+	It("should be schedule with cpu and memory limits", func() {
+		podClient := c.Pods(api.NamespaceDefault)
```

Comment:
nit: Needs to run in it's own namespace using @lavalamp's Framework.  But that's not how pods.go is wired up right now, so lets defer that to a separate PR.